### PR TITLE
PDK v1.18.1

### PR DESCRIPTION
**Pull request by yachub**
This updates the module to the latest (v1.18.1) PDK template

### CODE REVIEW

**Comment by yachub** [comment]
Hold off on this please. After performing a diff on the resulting nxlog.conf file of the `master` branch (v1.4.2) vs `develop` and `feature/pdk-1.18.1` of the module, it appears that the line ending of the file changed from CRLF to LF. I will investigate.

**Comment by genebean** [comment]
Hit me up in slack when you are ready for review, merge, and release.

**Comment by genebean** [comment]
Also, let's chat about killing off the develop branch.

File: .sync.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ---
+.gitattributes:
+  include:
+    '*.erb': eol=crlf
+    '*.epp': eol=crlf
 .gitlab-ci.yml:
   delete: true
 .travis.yml:
```
Comment (genebean) [inline .sync.yml]: Will you compare this one to https://github.com/ploperations/ploperations-webhook_proxy/blob/master/.sync.yml and see if there is more that should be done?

**Review by genebean — APPROVED** [review]

**Pull request merged**